updateengine/status.go

```diff
@@ -28,6 +31,55 @@ type Status struct {
 	NewSize          int64
 }
 
+var (
+	errStatusNil    = errors.New("dbus signal is nil")
+	errStatusLength = errors.New("signal body has insufficient length")
+	errStatusType   = errors.New("signal body has incorrect type")
```

Comment:
@lucab: do you think it would be worthwhile to format this error with the failing field?

Reply:
At the moment it is not really used, so probably not. If we need to surface that somewhere later, we can quickly do there.

Reply:
Will leave it for now. Would the failing field look something like `locksmith: error message here`?

Reply:
@rfairley it would be something like `fmt.Errorf("signal field %q has incorrect type", fieldname)` in `TryNewStatus` type checks.

Reply:
Right, I see. I think it can be left without specifying the field - I agree it can be resurfaced quickly if the need comes up.